recipes/arrow-c-glib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "arrow-c-glib" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: ad9a05705117c989c116bae9ac70492fe015050e1b80fb0e38fde4b5d863aaa3
+
+build:
+  number: 0
+  noarch: generic
+  skip: true  # [win]
+  missing_dso_whitelist:
```

Comment:
@xhochy not sure if this is the right way to go .. just added it in order to fix the issues on CI 
do you have any tips about this?